src/runtime/mod.rs

```diff
@@ -83,7 +94,21 @@ impl Runtime {
         })
     }
 
-    /// Runs a future to completion on the current runtime
+    /// Runs a future to completion on the tokio-uring runtime. This is the
+    /// runtime's entry point.
+    ///
+    /// This runs the given future on the current thread, blocking until it is
+    /// complete, and yielding its resolved result. Any tasks, futures, or timers
+    /// which the future spawns internally will be executed on this runtime.
+    ///
+    /// Any spawned tasks will be suspended after `block_on` returns. Calling
```

Comment:
@Noah-Kennedy needs to judge whether this statement about `block_on` is correct and whether it is correct to imply that `block_on` could be called more than once. Maybe it can and this is absolutely correct; I'm just saying I'm not sure myself

Reply:
It would be good to get a comment, but I based those docs on the tokio docs for block_on 
https://docs.rs/tokio/latest/tokio/runtime/struct.Runtime.html#current-thread-scheduler

Reply:
Again, if we don't hear from @Noah-Kennedy, you can just remove this change for now because while I like comments, I really care about technical comments that help us understand the tricky stuff when wanting to modify the design. The over all stuff about how to use it and how it relates to tokio I would prefer to leave to Noah over time.